src/index.js

```diff
@@ -173,7 +173,15 @@ registerPlugin(DragToScroll);
 registerPlugin(DropdownMenu);
 registerPlugin(ExportFile);
 registerPlugin(Filters);
+
+// TODO
+// Not messing with the bundling yet as I'm not sure if anything concrete
+// regarding that was decided so far.
+//
+// For now we'll just include it in the main bundle (which from what I see
+// doubles the size of the development bundle).
 registerPlugin(Formulas);
+
```

Comment:
We can revert this change, we've decided on Friday that Formulas will remain part of the bundle

Reply:
https://github.com/handsontable/handsontable/pull/7648/commits/8430a9a77fbf1d1dd85a3a6117968c3f6c56a069

Reply:
https://github.com/handsontable/handsontable/pull/7648/commits/8430a9a77fbf1d1dd85a3a6117968c3f6c56a069